packages/shaders/src/shaders/dot-grid.ts

```diff
@@ -111,7 +113,7 @@ void main() {
   vec2 p = (grid - center) * vec2(u_gridSpacingX, u_gridSpacingY);
 
   float baseSize = u_dotSize * (1. - sizeRandomizer * u_sizeRange);
-  float strokeWidth = u_strokeWidth;
+  float strokeWidth = u_strokeWidth * (1. - sizeRandomizer * u_sizeRange);
```

Comment:
Unrelated, but sneaking in a change/fix for dot grid stroke width to use the size range parameter too